index.html

```diff
@@ -17,4 +20,19 @@
     image_height: 13mm
     boder width: 1mm
   -->
+  <div class="wrapper-print-stamp">
+    <img src="./assets/images/logo.jpg" alt="Logo" class="logo">
+    <div class="print-stamp-title text-bold">
+      <span>NATURAL JADEITE</span> <br />
+      <span>JADE</span> <br />
```

Comment:
```suggestion
      <span>NATURAL JADEITE JADE</span> <br />
```
gộp lại trong 1 span
tự xuống huống nếu bị overflow-x